conda/recipe/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "open-ce" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - git_url: ../..
+
+build:
+  number: 1
```

Comment:
I'm not sure we want to explicitly add a build number here. I don't want to have to update the build number every time we make a change to the open-ce repo. Maybe there is something else we can use like the GitHub PR number.

Reply:
In that case we would even want to remove github workflow tests that should execute `check_build_numbers.py` on this recipe.

Reply:
@bnemanich - For build number I've used `GIT_DESCRIBE_NUMBER` which is number of commits since the last tag created on the main branch. So, after every commit/PR merge to main branch it will keep on incrementing until we create a tag. Once the new tag is created, it will be back to 0 and again start incrementing. I think this is the better option than using Github PR number (which can be parsed from $GITHUB_REF) as this variable is empty if we locally try to use it.

There is another side problem that I'm seeing with `git describe --tags` on our open-ce main branch. It shows `open-ce-v1.1.2-38-g6f5c26b` whereas it should have been `open-ce-v1.1.3-35-...`. It could be because 1.1.3 is not tagged from main branch. (_38 is the commit count since 1.1.2 and 35 since 1.1.3 on the main branch_)